src/Runner.Worker/ActionCommandManager.cs

```diff
@@ -563,13 +594,67 @@ public void ProcessCommand(IExecutionContext context, string inputLine, ActionCo
             context.AddIssue(issue);
         }
 
+        public static bool ValidateLinesAndColumns(ActionCommand command, IExecutionContext context) 
+        {
+            command.Properties.TryGetValue(IssueCommandProperties.Line, out string line);
+            command.Properties.TryGetValue(IssueCommandProperties.EndLine, out string endLine);
+            command.Properties.TryGetValue(IssueCommandProperties.Column, out string column);
+            command.Properties.TryGetValue(IssueCommandProperties.EndColumn, out string endColumn);
```

Comment:
do we need to consider cases where `endColumn` != empty string but int.tryparse fails?

Reply:
This is accounted for since `int.TryParse` will return false if it can't parse an int